Scripts/SL-Histogram.lua

```diff
@@ -96,7 +96,13 @@ function NPS_Histogram(player, width, height)
 		InitCommand=function(self)
 			self:SetDrawState({Mode="DrawMode_QuadStrip"})
 		end,
-		CurrentSongChangedMessageCommand=function(self)
```

Comment:
I'm not sure if CurrentSongChangedMessageCommand is still necessary for course mode.

Reply:
I just checked and it seems like CurrenSongChangedMessageCommand is not necessary for course mode :)